rpc/src/v1/tests/mocked/ethcore.rs

```diff
@@ -288,6 +291,21 @@ fn rpc_ethcore_unsigned_transactions_count_when_signer_disabled() {
 }
 
 #[test]
+fn rpc_ethcore_hash_content() {
```

Comment:
bit iffy - generally best not to rely upon external web servers for unit tests. probably fine for now but should eventually replace with something more certain.